input/pagecontent/index.md

```diff
@@ -1,44 +1,62 @@
+Conceived in September 2019 and initially released in February 2020, FHIR Shorthand (FSH) has been rapidly adopted by the FHIR community. The [FSH Finder](https://fshschool.org/fsh-finder/) tool reports well over 100 projects using FSH, and [npm-stat](https://npm-stat.com/charts.html?package=fsh-sushi&from=2020-02-12&to=2021-07-12) shows that the *fsh-sushi* [npm](https://www.npmjs.com/) package has had over 45,000 downloads as of July 2021. The [#shorthand channel on chat.fhir.org](https://chat.fhir.org/#narrow/stream/215610-shorthand) has 270 subscribers and participation is very active. FSH and related tools have been tested at three HL7 Connectathons. Several significant tools for processing FSH have been developed, including [SUSHI](https://fshschool.org/docs/sushi/), a reference implementation for transforming FSH into FHIR artifacts, and [GoFSH](https://fshschool.org/docs/gofsh/), a tool for transforming FHIR artifacts to FSH. Using these tools, FSH can be losslessly round-tripped to FHIR JSON and back. FSH and SUSHI have been integrated with the [HL7 FHIR Implementation Guide Publishing tool](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation), allowing seamless processing from FSH to a complete IG.
 
-1. [SUSHI User Guide](https://fshschool.org/docs/sushi/) -- A guide to producing an IG from FSH files using SUSHI compiler and the HL7 IG Publishing tool.
-1. [FHIR Shorthand Tutorials](https://fshschool.org/docs/tutorials/) -- A step-by-step hands-on introduction to producing an Implementation Guide (IG) with FHIR Shorthand and SUSHI.
+FSH was approved as a Standard for Trial Use (STU 1) in May 2020. In the ensuing period, vigorous activity around FSH has driven improvements, new features, and rapid maturation of FSH and related tools. As a result, the FSH specification is now being offered for a mixed normative/trial use ballot. The majority of language features of FSH are proposed as normative, including some post-STU 1 features that have benefited from significant user testing. Certain new language features, such as defining logical models, are proposed as Trial Use. Trial use features are clearly marked in the [language specification](reference.html) with the {%include tu.html%} symbol.
 
+Designation as a normative standard does not mean that the FSH language will not continue to evolve, only that any future changes must be compatible with the normative portions of the specification.
 
-### Notational Conventions
+#### Motivations for FHIR Shorthand
 
-This IG uses the following conventions:
+FSH was created in response to the need in the FHIR community for scalable, fast, user-friendly tools for IG creation and maintenance. Experience has shown that profiling projects can be difficult and slow, and the resulting IG quality inconsistent. Profiling projects often go through many iterations. As such, an agile approach to refactoring and revision is invaluable.
```

Comment:
There is a lot of passive voice in the above paragraph.  Consider rewording to use active voice, putting the author in the center.  Perhaps something like:
```suggestion
FSH was created in response to the need in the FHIR community for scalable, fast, user-friendly tools for IG creation and maintenance. IG authors often struggle to implement profiling projects with efficiency, consistency, and quality. Project teams iterate over the formal definitions and examples many times during the IG development process. As such, an agile approach to refactoring and revision is invaluable.
```